README.md

```diff
@@ -13,8 +13,42 @@ You may install it yourself with:
 
 ## Usage
 
-This is still a Work in Progress and is not intended for public usage yet.
-Use at your own risk!
+For initial CA setup, we provide two options. These need to be run before starting
+Puppet Server for the first time.
+
+To set up a default CA, with a self-signed root cert and an intermediate signing cert:
+```
+puppetserver ca setup
+```
+
+To import a custom CA:
+```
+puppetserver ca import --cert-bundle certs.pem --crl-chain crls.pem --private-key ca_key.pem
+```
+
+The remaining actions provided by this gem require a running Puppet Server, since
+it primarily uses the CA's API endpoints to do its work. The following examples
+assume that you are using the gem packaged within Puppet Server.
+
+To sign a pending certificate request:
+```
+puppetserver ca sign --certname mycsr.pem
```

Comment:
This is actually wrong, it should `--certname foo.example.com` or something that doesn't look like a filepath.

Reply:
Oh, derp.